openai/openai.go

```diff
@@ -23,6 +23,10 @@ const (
 		If the problem is a request for clarification, prefix your comment with "Clarification:".
 		If a unit test of critical functionality is missing, prefix your comment with "Missing Test:".
 		If a unit test could use some more test cases, prefix your comment with "Suggested Test Cases:".
+		If a method or class is too big, prefix your comment with "Refactor Suggestion:".
+		If you see lots of duplicated code, prefix your comment with "Refactor Suggestion:".
+		If a function or variable could be named more descriptively, prefix your comment with "Suggestion:".
+		Comment on all "TODO" comments.
 		Copy the "+" or "-" into your comment prefix before the line number. 
 		Only rarely comment on a line that starts with "-".
```

Comment:
The printTokenUsage function is only used within the openai package, consider moving it to the same file as its only caller.

Reply:
it is already in the same file.